### DISCUSSION TITLE
Building/running images from docker-compose

### DISCUSSION THREAD

**Discussion by accetto** [accetto/ubuntu-vnc-xfce-g3#42] [Ideas]
This thread replaces the original thread #39 (opened by @capsulecorplab), which has been renamed to `approach to binding a directory (from host machine) to containers $HOME directory`.

**Comment by accetto** [comment]
The new [User guide](https://accetto.github.io/user-guide-g3/) contains a section about using Docker Compose with my images. First examples are already included and more are still to come. Therefore I'm closing this discussion. Please use the [Issues](https://github.com/accetto/user-guide-g3/issues) of the User guide project for feedback.

**Reply by capsulecorplab** [reply]
Awesome! I'll have to give this a shot, hopefully sometime this coming week
